docs/topics/api-guidelines-build-for-multiplatform.md

```diff
@@ -0,0 +1,84 @@
+[//]: # (title: Building a Kotlin library for multiplatform)
+
+When creating a Kotlin library, consider building and [publishing the library with support for Kotlin Multiplatform](multiplatform-publish-lib.md).
+This broadens the target audience of your library, making it compatible with projects targeting multiple platforms.
+
+The following sections provide guidelines to help you build a Kotlin Multiplatform library effectively.
+
+## Maximize your reach
+
+To make your library available to the largest number of projects as a dependency,
+aim to support as many [target platforms](multiplatform-dsl-reference.md#targets) of Kotlin Multiplatform as possible.
+If your library doesn’t support the platforms used by a multiplatform project,
+whether it’s a library or an application, it becomes difficult for that project to depend on your library.
+Projects might use your library for some platforms and implement separate solutions for others,
+or choose an alternative library altogether that supports all their platforms.
+
+> For Kotlin/Native targets, consider using a [tiered approach](native-target-support.md#for-library-authors) to support all possible targets.
+>
+{style="note"}
+
+## Design APIs for use from common code
+
+When creating a library, design the APIs to be usable from common Kotlin code, instead of writing platform-specific code.
```

Comment:
```suggestion
When creating a library, design the APIs to be usable from common Kotlin code instead of writing platform-specific implementations.
```

Reply:
double 'code'